Editor/AGS.Editor/Resources/agsdefns.sh

```diff
@@ -2471,6 +2529,36 @@ builtin struct Screen {
 };
 #endif
 
+#ifdef SCRIPT_API_v360

```

Comment:
I think you missed to update this to "SCRIPT_API_v399"

Reply:
ok, fixed this!